x/project/handler.go

```diff
@@ -227,7 +268,21 @@ func handleMsgCreateClaim(ctx sdk.Context, k Keeper, fk payments.Keeper,
 		return err.Result()
 	}
 
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeUpdateProjectStatus,
+			sdk.NewAttribute(types.AttributeKeyTxHash, msg.TxHash),
+			sdk.NewAttribute(types.AttributeKeySenderDid, msg.SenderDid),
+			sdk.NewAttribute(types.AttributeKeyProjectDid, msg.ProjectDid),
+			sdk.NewAttribute(types.AttributeKeyData, msg.Data.ClaimID),
```

Comment:
Please add `msg.Data.Status` as an attribute. And then the attribute keys would be: `AttributeKeyClaimID` and `AttributeKeyClaimStatus`.

Reply:
AttributeKeyClaimStatus did not find..
type CreateClaimDoc struct {
	ClaimID string `json:"claimID" yaml:"claimID"`
}